database/migrations/sqls/20210219043110-tree-up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE tree
+(
+    id uuid NOT NULL PRIMARY KEY,
+    latest_capture_id uuid NOT NULL,

```

Comment:
I am not sure if this naming is adequate. `latest_capture_id` would reference the last matched capture to this tree.